### PR TITLE
Highlight current tab in header, add pause play capability to slideshow

### PR DESCRIPTION
**Pull request by IshaniSanturkar**
In this pull request I, 
- added styling to highlight the current tab you are on, for example Projects or About Me, in a darker shade in the header. 
- modified the next and previous buttons in the slideshow to be arrows on the slideshow itself.
- added the capability to pause and resume the slideshow by clicking on the window. 

The current state of the website is: [portfoliov4.zip](https://github.com/IshaniSanturkar/step/files/4703089/portfoliov4.zip)

### CODE REVIEW

**Comment by EatingW** [comment]
HI Ishani, would you like to pull master and resolve merge conflicts first? I'll review after!

File: portfolio/src/main/webapp/gallery.html [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
-      <input type="range" min="1" max="10" value="5" class="slider" id="galleryslider" onchange="sliderMoved()">
-    </div>
-    </div>
+  <head>
+    <meta charset="UTF-8">
+    <title>Gallery</title>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+     <script src="script.js"></script>
```
Comment (EatingW) [inline portfolio/src/main/webapp/gallery.html]: nit: delete space

File: portfolio/src/main/webapp/gallery.html [hunk 2]
```diff
@@ -1,35 +1,34 @@
...
+  </head>
+  <body id="content">
+     <header>
```
Comment (EatingW) [inline portfolio/src/main/webapp/gallery.html]: nit: indent 2 spaces. Same for the sections below

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -35,50 +36,91 @@ function toggleCourseList(listObject) {
...
+    statusImg.src = "/images/play.png";
+    statusImg.style.display = "block";
+    window.setTimeout(function () {
+      $("#pauseplay").fadeOut();
+      statusImg.style.display = "none";
+    }, 250);
+    myTimer = setInterval(loopOverImages, sliderSpeed);
+    paused = false;
+  } else {
+    console.log("Hi");
```
Comment (EatingW) [inline portfolio/src/main/webapp/script.js]: nit: left over from debugging?

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -35,50 +36,91 @@ function toggleCourseList(listObject) {
...
+    clearInterval(myTimer);
+    const statusImg = document.getElementById("pauseplay")
+    statusImg.src = "/images/pause.png";
+    statusImg.style.display = "block";
+    window.setTimeout(function () {
+      $("#pauseplay").fadeOut();
+      statusImg.style.display = "none";
+    }, 500);
```
Comment (EatingW) [inline portfolio/src/main/webapp/script.js:118]: Is there a reason that the play fadeout time and pause fadeout time are different?

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -35,50 +36,91 @@ function toggleCourseList(listObject) {
...
+  }
   const sliderValue = document.getElementById("galleryslider").value;
   sliderSpeed = sliderValue * 1000;
-  myTimer = setInterval(loopOverImages, sliderSpeed);
+  if (!paused) {
+    myTimer = setInterval(loopOverImages, sliderSpeed);
+  }
+}
+
+
```
Comment (EatingW) [inline portfolio/src/main/webapp/script.js]: nit: one linebreak between functions

**Review by EatingW — COMMENTED** [review]
LGTM modulo some minor nits!

For the icons and CMU-related images, please make sure that they are free for you to use if you didn't create them! If you're not sure about licensing, I would suggest asking the VSE team or the intern team.

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (IshaniSanturkar) [reply portfolio/src/main/webapp/script.js:118]: Accidental, sorry.

**Review by EatingW — APPROVED** [review]
LGTM!

**Pull request merged**